### PR TITLE
TECH-649: Create cucumber test for POST /linked-authorization/link-transaction

### PR DESCRIPTION
**Pull request by KarolinaKopacz**
Create cucumber test for POST /linked-authorization/link-transaction

Ticket: https://govstack-global.atlassian.net/browse/TECH-649

### CODE REVIEW

File: test/openAPI/features/support/wallet_link_transaction.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  }
+);
+
+When(
+  'Send POST \\/linked-authorization\\/link-transaction request with given linkCode',
+  () =>
+    specWalletLinkTransaction.post(baseUrl).withJson({
+      requestTime: new Date().toISOString(),
+      request: {
+        linkCode: recivedLinkCode,
```
Comment (mdulko) [inline test/openAPI/features/support/wallet_link_transaction.js:60]: please fix 'recived' to 'received'

**Review by mdulko — CHANGES_REQUESTED** [review]

**Review by mdulko — APPROVED** [review]
lgtm

File: test/openAPI/features/wallet_link_transaction.feature [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  @smoke @unit @positive
+  Scenario: Successfully validates the link-code and its expiry and generates the linkTransactionId smoke type test
+    Given Wants to validate the link-code and its expiry and generate the linkTransactionId
+    And The link code is generated before POST /linked-authorization/link-transaction
+    When Send POST /linked-authorization/link-transaction request with given linkCode
+    Then Receive a response from the /linked-authorization/link-transaction endpoint
+    And The /linked-authorization/link-transaction endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/link-transaction endpoint response should have status 200
+    And The /linked-authorization/link-transaction endpoint response should have content-type: application/json header
+    And The /linked-authorization/link-transaction endpoint response should match json schema
```
Comment (vishwa-vyom) [inline test/openAPI/features/wallet_link_transaction.feature:13]: should this not be "match json schema with no error" to be consistent with other endpoint feature files ?

File: test/openAPI/features/wallet_link_transaction.feature [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  @negative
+  Scenario: Not able to validate the link-code and its expiry and generate the linkTransactionId because of invalid linkCode
+    Given Wants to validate the link-code and its expiry and generate the linkTransactionId
+    When Send POST /linked-authorization/link-transaction request with given invalid linkCode
+    Then Receive a response from the /linked-authorization/link-transaction endpoint
+    And The /linked-authorization/link-transaction endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/link-transaction endpoint response should have status 200
+    And The /linked-authorization/link-transaction endpoint response should have content-type: application/json header
+    And The /linked-authorization/link-transaction endpoint response should match json schema
+    And The /linked-authorization/link-transaction response should contain errorCode property equals to "invalid_link_code"
```
Comment (vishwa-vyom) [inline test/openAPI/features/wallet_link_transaction.feature:24]: Can we also add one more negative scenario to send expired linkCode and expect "invalid_link_code"

**Review by vishwa-vyom — COMMENTED** [review]
Reviewed feature file and added comments

File: test/openAPI/features/wallet_link_transaction.feature (hunk 2 above)
Comment (KarolinaKopacz) [reply test/openAPI/features/wallet_link_transaction.feature:13]: Good point!

File: test/openAPI/features/wallet_link_transaction.feature (hunk 3 above)
Comment (KarolinaKopacz) [reply test/openAPI/features/wallet_link_transaction.feature:24]: I added new scenario in https://github.com/GovStackWorkingGroup/bb-identity/pull/40 PR which also includes fixes after your comments.

Please continue discussion in the new linked PR :)

Comment (vishwa-vyom) [reply test/openAPI/features/wallet_link_transaction.feature:24]: Added comments #40, so marking this as resolved.

**Review by vishwa-vyom — COMMENTED** [review]
All comments are moved to #40 marking review on this as complete

**Pull request merged**